Clava-JS/src-api/clava/events/EventListener.ts

```diff
@@ -0,0 +1,141 @@
+import { EventEmitter } from "events";
+import { Event, EventTime } from "./Events.js";
+import ophistory from  "../history/History.js"
+import { DetachOperation, DetachReference, InlineCommentOperation, InsertOperation, RemoveChildrenOperation, ReplaceOperation, SetChildOperation, TypeChangeOperation, ValueOperation } from "../history/Operations.js";
+import { Joinpoint } from "../../Joinpoints.js";
+
+const eventListener = new EventEmitter();
+
+// Used for saving previous child in setFirstChild and setLastChild
+let auxJP: Joinpoint;
+
+eventListener.on("ACTION", (e: Event) => {
```

Comment:
These are the events that store the necessary data to create an history of operations, probably this code belongs to the history package, possibly the Operations file, or could be on its own OperationEvents file.

Reply:
Fixed in [this commit](https://github.com/specs-feup/clava/pull/157/commits/c363c12293e743e758cb9d9db812b542a3f9ed02)